omod/src/test/webapp/resources/scripts/navigatorHandlers.js

```diff
@@ -784,7 +788,7 @@ describe("Tests for simple form navigation handlers", function() {
                 clickedQuestionHandler(questions, secondQuestion, event);
 
                 expect(event.stopPropagation).toHaveBeenCalled();
-                expect(field.select).toHaveBeenCalled();
```

Comment:
I wouldn't expect this to be failed now, was it?

Reply:
I added it back.